tests/PropertySpec/Exec.hs

```diff
@@ -0,0 +1,47 @@
+module PropertySpec.Exec
+  ( additionSpec
+  , subtractionSpec
+  , multiplicationSpec
+  , divisionSpec
+  )where
+
+import           Data.Foldable                               (fold, foldl1)
+import           Data.List.NonEmpty                          (intersperse)
+import           ExecSpec.Util                               (num)
+import           Graphics.Implicit.ExtOpenScad.Definitions   (OVal (ONum))
+import           Graphics.Implicit.ExtOpenScad.Eval.Constant (runExpr)
+import           HaskellWorks.Hspec.Hedgehog                 (requireProperty)
+import           Hedgehog                                    (diff, forAll)
+import qualified Hedgehog.Gen                                as Gen
```

Comment:
still need to fully qualify imports.

Reply:
I don't think that it is an issue, as you always know where the qualified function call comes from up to overloading the qualified namespace.

Reply:
Sure, but it's harder to see what in the import is being used, without digging through the whole file. I find it useful when dealing with larger libraries to look through their import lists for functions in a similar vein, and standardize on using less functions.